agences-regionales-sante/nouvelle-aquitaine/2020-03-26.yaml

```diff
@@ -0,0 +1,14 @@
+date: 2020-03-26
+source:
+  nom: ARS Nouvelle-Aquitaine
+  url: https://www.nouvelle-aquitaine.ars.sante.fr/system/files/2020-03/CP_Covid19_ARSNA_26_03_2020_0.pdf
+  archive: https://web.archive.org/web/20200327143008/https://www.nouvelle-aquitaine.ars.sante.fr/system/files/2020-03/CP_Covid19_ARSNA_26_03_2020_0.pdf
```

Comment:
archive is down!
I generate one: https://web.archive.org/web/20200327184143/https://www.nouvelle-aquitaine.ars.sante.fr/system/files/2020-03/CP_Covid19_ARSNA_26_03_2020_0.pdf

Reply:
Merci pour le lien !
J'ai aussi ajouté l'heure des données (14h SPF Nouvelle-Aquitaine).

Reply:
Yep, souvnt pas facile a voir.